migrations/Version20240310150000.php

```diff
@@ -0,0 +1,671 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Platforms\MySQLPlatform;
+use Doctrine\DBAL\Platforms\PostgreSQLPlatform;
+use Doctrine\DBAL\Platforms\SqlitePlatform;
+use Doctrine\DBAL\Schema\Schema;
+use Wallabag\Doctrine\WallabagMigration;
+
+final class Version20240310150000 extends WallabagMigration
+{
+    public function getDescription(): string
+    {
+        return 'Fix schema';
+    }
+
+    public function up(Schema $schema): void
+    {
+        switch (true) {
+            case $this->platform instanceof MySQLPlatform:
+                $this->write('Align database schema with Doctrine metadata.');
+
+                $this->addSql('ALTER TABLE ' . $this->getTable('oauth2_clients') . ' CHANGE name name LONGTEXT NOT NULL;');
+
+                $this->addSql('ALTER TABLE ' . $this->getTable('user') . ' CHANGE salt salt VARCHAR(255) DEFAULT NULL;');
```

Comment:
In which cases salt is empty?

Reply:
I don't know, change is to align with [FOSUserBundle mapping](https://github.com/FriendsOfSymfony/FOSUserBundle/blob/master/Resources/config/doctrine-mapping/User.orm.xml#L19), see the `nullable="true"`